jdaviz/configs/cubeviz/plugins/parsers.py

```diff
@@ -404,7 +444,9 @@ def _parse_ndarray(app, file_obj, data_label=None, data_type=None,
 
     if not hasattr(flux, 'unit'):
         flux = flux << u.count
-    s3d = Spectrum1D(flux=flux)
+
+    meta = standardize_metadata({'_orig_spatial_wcs': None})
```

Comment:
I am not sure if `standardize_metadata` is needed for simple cases like this but I guess some consistency does not hurt.